__tests__/app.test.js

```diff
@@ -166,6 +166,29 @@ describe("GET articles sort queries", ()=>{
             expect(article).toHaveProperty("body")
             expect(article).toHaveProperty("topic")
 
+            expect(article).toHaveProperty("created_at")
+            expect(article).toHaveProperty("votes")
+            expect(article).toHaveProperty("article_img_url")
+            })
+        })
+    })
+});
+
+describe("GET articles by topic query", ()=>{
+    test("Return all the available articles filtered by topic",()=>{
+        return request(app)
+        .get("/api/articles?topic=mitch")
+        .expect(200)
+        .then((responce) => {
+            expect(Array.isArray(responce.body.articles)).toBe(true);
+            responce.body.articles.forEach((article)=>{
```

Comment:
This test does not tell me that you have filtered the articles, so only the ones about `mitch` return. We need to check the `articles.length` is what we expect it to be.